### ISSUE TITLE
iPhone 5s device did not reconnect

### ISSUE THREAD

**Issue by pixdoet** [LukeZGD/iOS-OTA-Downgrader#171]
[Output log](https://pastebin.com/uRqa0H8V)  

Hi, I have been trying to use this tool to downgrade my iPhone 5s. However, for every attempt I run the script, it exits with this error: `Device did not reconnect. Possibly invalid iBSS. Reset device and try again`. 

I have resetted the device multiple times, and have also tried different usb cables and ports (none of them are USB-C to Lightning btw) and it still doesn't work. 

Do please help. Thanks!

**Comment by LukeZGD** [comment]
I have seen this issue reported quite a few times, I'm not really sure on exactly how to solve this. The most recent similar issue is #168 (though it's iBEC, not iBSS), you may check out the thread

You may also try selecting ipwnder_lite as pwnDFU tool, though I haven't really tested this myself

**Comment by pixdoet** [comment]
Alright, I'll try with ipwnder_lite and see if it works. Thanks!

**Comment by pixdoet** [comment]
Hi, I've tried with ipwnder_lite and it still shows the same error. Are there any other things that I can try out?

**Comment by LukeZGD** [comment]
Try my suggestions in issue #168 

Also the OP of the said issue also did some procedure (not sure how it works) and that worked for them

**Comment by tobia** [comment]
I'm from issue #168. My problem was that a different (older) downgrading script (that I had tried before finding this one) had put my iPad in a weird state, where it would boot into recovery mode (or DFU, depending on the button combinations) but then both this script and the official Finder restore button would fail to flash it.

I tried both this script and the Finder restore button dozens of times, on different Mac and Linux PCs, with different cables, to no avail. I finally solved the problem by doing the following procedure that I found randomly online:

1. turn off the device
2. plug a headphone into the 3.5mm jack
3. plug the USB cable into a Mac
4. the device will turn on by itself into recovery mode (iTunes icon)
5. place it into DFU mode (you can use the beginning of iOS-OTA-Downgrader as a guide for which buttons to press for how long)
6. as soon as it enters DFU mode, quit iOS-OTA-Downgrader and use the official Restore button in Finder
7. in my case, this restored the device to the latest OS and made it usable again
8. then I proceeded to run iOS-OTA-Downgrader again and this time it worked.

I have no idea what the headphone trick does (if anything) but in my case I had tried several times and always failed, until I used it.

**Comment by LukeZGD** [comment]
I'll be closing the issue now since the problem is not really with the scripts. OP, give the procedure a try, it may help with a successful restore

**Comment by voidf1sh** [comment]
Sorry to bump an old issue but I wanted to add the solution for me. I was attempting to downgrade my iPad Mini 2 (A1498, iPad4,4) to 10.3.3 on an M1 MacBook on Ventura 13.2.

I had to use a USB-A to Lightning cable and a USB-A to USB-C adapter to get the iPad into DFU mode, and I had to use a USB-C to Lightning cable to do the rest. I also had to disconnect and reconnect while iBSS was waiting for a reconnect. After that there were a couple prompts to allow an accessory to connect to the mac, once approved everything went off without a hitch.

Thank you @LukeZGD for this awesome tool!
